clippy_lints/src/methods.rs

```diff
@@ -494,13 +494,33 @@ declare_lint! {
 /// s.push_str(abc);
 /// s.push_str(&def));
 /// ```
-
 declare_lint! {
     pub STRING_EXTEND_CHARS,
     Warn,
     "using `x.extend(s.chars())` where s is a `&str` or `String`"
 }
 
+/// **What it does:** Checks for the use of `.cloned().collect()` on slice to create a Vec.
+///
+/// **Why is this bad?** `.to_owned()` is clearer
+///
+/// **Known problems:** None.
+///
+/// **Example:**
+/// ```rust
+/// let s = [1,2,3,4,5];
+/// let s2 : Vec<isize> = s.iter().cloned().collect();
+/// ```
+/// The correct use would be:
```

Comment:
s/The correct/A better

“The correct” implies it's wrong, but it's only bad :smile: